File: examples/id/generate/node/generateAUniqueId.js
import * as id from "@m3o/services/id";

// Generate a unique ID. Defaults to uuid.
async function GenerateAuniqueId() {
  let idService = new id.IdService(process.env.MICRO_API_TOKEN);
  let rsp = await idService.generate({
    type: "uuid",
  });
  console.log(rsp);
}

await GenerateAuniqueId();

File: examples/id/generate/go/generateAUniqueId.go
package example

import (
	"fmt"
	"github.com/micro/services/clients/go/id"
	"os"
)

// Generate a unique ID. Defaults to uuid.
func GenerateAuniqueId() {
	idService := id.NewIdService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := idService.Generate(&id.GenerateRequest{
		Type: "uuid",
	})
	fmt.Println(rsp, err)
}
